server/engine/src/main/antlr4/org/eclipse/lsp/cobol/implicitDialects/cics/CICSLexer.g4

```diff
@@ -1708,6 +1708,112 @@ ZCPTRACING : Z C P T R A C I N G;
 ZERO : Z E R O;
 ZEROES : Z E R O E S;
 ZEROS : Z E R O S;
+NORMAL : N O R M A L;
```

Comment:
Should this be added also into the `CICS.tmLanguage.json`?

Also please re-sort all the lists (not-just in this file).